cypress/integration/account_settings/display/code_theme_colors_spec.js

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/*eslint max-nested-callbacks: ["error", 3]*/
+
+const THEMES = [{name: 'github', backgroundColor: 'rgb(248, 248, 248)', color: 'rgb(51, 51, 51)'},
+    {name: 'monokai', backgroundColor: 'rgb(39, 40, 34)', color: 'rgb(221, 221, 221)'},
+    {name: 'solarized-light', backgroundColor: 'rgb(253, 246, 227)', color: 'rgb(101, 123, 131)'},
+    {name: 'solarized-dark', backgroundColor: 'rgb(0, 43, 54)', color: 'rgb(131, 148, 150)'},
+];
+
+function verifyLastPostStyle(codeTheme) {
+    cy.getLastPostId().then((postId) => {
+        const postCodeBlock = `#postMessageText_${postId} code`;
+
+        // * Verify that the code block background color and color match the desired theme
+        cy.get(postCodeBlock).
+            should('have.css', 'background-color', codeTheme.backgroundColor).
+            and('have.css', 'color', codeTheme.color);
+    });
+}
+
+function navigateToThemeSettings() {
+    // Change theme to desired theme (keeps settings modal open)
+    cy.toAccountSettingsModal(null, true);
+    cy.get('#displayButton').click();
+    cy.get('#displaySettingsTitle').should('exist');
+
+    // Open edit theme
+    cy.get('#themeTitle').should('be.visible');
+    cy.get('#themeEdit').click();
+    cy.get('.section-max').scrollIntoView();
+}
+
+describe('M14319 Display Settings > Custom Theme > Code Theme', () => {
```

Comment:
Please change description to match GH/Jira ticket like "AS14319 Theme Colors - Code".

Reply:
@saturninoabril Description has been updated.